tests/spec/core/data-tests-spec.js

```diff
@@ -63,5 +68,33 @@ describe("Core — data-tests attribute", () => {
         expect(pathname.endsWith(`test${i}.html`)).toBe(true);
       });
     });
+    it(`adds emojis for manual and https related tests`, () => {
+      
+      let test = doc.querySelector("#metadata-tests ul").querySelectorAll("li")
```

Comment:
As you have the tests here,  we should split the tests into three separate `it()` statements:

```JS
it("adds emojis for secure tests", () => {
  const li = doc.querySelector("#metadata-tests ul>li:nth-child(1)");
  // other stuff
});
it("adds emojis for manual tests", () => {
  const li = doc.querySelector("#metadata-tests ul>li:nth-child(2)");
  // other stuff
});
it("adds emojis for secure manual tests", () => {
  const li = doc.querySelector("#metadata-tests ul>li:nth-child(3)");
  // other stuff
});
```